src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-big-one">
+      <form
+        action=""
```

Comment:
I get 405 page when I press Enter, you should fix your action value
![image](https://user-images.githubusercontent.com/37161315/137089386-9f4f887f-cc91-4d66-814c-1a7972cf9a8a.png)